q2_diversity/plugin_setup.py

```diff
@@ -303,6 +303,14 @@
     name='Core diversity metrics (phylogenetic and non-phylogenetic)',
     description="Applies a collection of diversity metrics (both "
                 "phylogenetic and non-phylogenetic) to a feature table.",
+    citations=[
+        citations['lozupone2005unifrac'],
+        citations['lozupone2007quantitative'],
+        citations['faith1992conservation'],
+        citations['vazquez2013emperor'],
+        citations['jaccard1908nouvelles'],
+        citations['pielou1966measurement'],
+        citations['sorensen1948method']]
```

Comment:
I hate to say this after all your work, but the pipelines won't typically need citations on their own, as the actions they call are tracked in provenance (and so are their citations). Pipelines only need references if their doing something like open-reference.

Reply:
oh right of course! No problem — the pipelines are the minority, and I just copied/pasted the other citations so it really didn't add work.